app/src/main/java/nu/yona/app/api/manager/network/BaseImpl.java

```diff
@@ -253,9 +253,9 @@ void onError(retrofit2.Response response, DataLoadListener listener)
 				Converter<ResponseBody, ErrorMessage> errorConverter =
 						getRetrofit().responseBodyConverter(ErrorMessage.class, new Annotation[0]);
 				ErrorMessage errorMessage = errorConverter.convert(response.errorBody());
-				if (errorMessage != null && errorMessage.getCode() != null)
+				if (errorMessage != null)
```

Comment:
its because of this `YonaActivity.getActivity().showError(new ErrorMessage(getString(R.string.no_friend_text)));`

Reply:
Then the code could be null, but the error message it self not, right?

Reply:
In this case errorMessage setting doesnt set the code. Earlier code used to be empty string, but now it will be null. 
Every other place code is written to check for errormessage.getcode for null, but here it was not there, so it started to crash while showing the `no friends to` show toast

Reply:
> Can this ever be null?
Yes. this can not be null. I am in doubt of some corner case scenarios we miss while testing  and where response body could not be properly converted to errorMessage.

Reply:
Then we should correct that. There is no reason why an arbitrary response cannot be converted to an error message.

Reply:
ok. I will remove the check.